lib/integration.js

```diff
@@ -51,6 +53,32 @@ function testIntegration(callback) {
 }
 
 
+/**
+ * Run an integration test using configuration file.
+ */
+function testIntegrationFile(callback) {
+	const server = testserver.startServer({ port: PORT }, error => {
+		if (error) {
+			return callback(error);
+		}
+		execFile('node',
+			[join(__dirname, '..', 'bin', 'loadtest.js'), 'http://localhost:' + PORT],
+			{ cwd: join(__dirname, '..', '__tests__') },
```

Comment:
I cannot see where you are specifying the name of the config file?

Reply:
The name of the configuration file is not specified (using default), only the directory where the search should start.